packages/browser-wallet/src/popup/pages/Account/i18n/en.ts

```diff
@@ -74,9 +74,13 @@ const t = {
         add: {
             lookupTokens: 'Look for tokens',
             indexRequired: 'Contract index is required',
+            negativeIndex: 'Contract index may not be negative',
```

Comment:
May not be makes it sounds like a suggestion. Perhaps it should be more firm like:
```
Contract index must be non-negative
```
or something along those lines. Perhaps Danielle has a better phrasing here.

Reply:
I'd think we want to avoid non-negative :thinking: 
I've changed it to "...index cannot be negative" to make it sound less suggestive.